lib/local_daily_digest.js

```diff
@@ -30,18 +39,61 @@ function get(masterCb) {
 				lastDigestSent = new Date(lastDigestSent);
 				var offset = current.valueOf() - lastDigestSent.valueOf();
 				if (offset > 24 * 60 * 60 * 1000) { // If last digest was sent more than 24 hours ago
+					//if profile roles does not contain manager or editor
 	  				allLocalProfilesWDigest.push(profile);
+	  			
+	  				if(!(matchInArray('manager:', profile.roles)) && !(matchInArray('editor:', profile.roles)))
```

Comment:
I don't think so. I think it should be: foreach location in the dailyDigest array { if user is a manager or editor OF THIS LOCATION { send the full digest } else { send the org digest } }